src/Analyzers/Core/Analyzers/IDEDiagnosticIds.cs

```diff
@@ -154,6 +154,7 @@ internal static class IDEDiagnosticIds
 
         public const string RemoveUnnecessaryDiscardDesignationDiagnosticId = "IDE0110";
 
+        public const string SimplifyLinqExpressionsDiagnosticId = "IDE0085";
```

Comment:
0085?

Reply:
moved this nearer to `0084` I can skip that gap if you like